js-core-1/week-2/lesson.md

```diff
@@ -237,262 +271,133 @@ function satisfiesRequirements(num) {
+let mentor1 = "Daniel";
+let mentor2 = "Irina";
+let mentor3 = "Rares";
 ```
 
 ...then it's probably time to use an **array**!
 
 Arrays are data structures that hold a list of values.
 
 ```js
-var mentors = ["Daniel", "Irina", "Rares"];
+let mentors = ["Daniel", "Irina", "Rares"];
 ```
 
 Arrays can hold any type of value (although almost always you only have one data type per array).
 
 ```js
-var testScores = [16, 49, 85];
-var grades = ["F", "D", "A"];
-var greetings = ["Hello, how are you?", "Hi! Nice to meet you!"];
+let testScores = [16, 49, 85];
+let grades = ["F", "D", "A"];
+let greetings = ["Hello, how are you?", "Hi! Nice to meet you!"];
 ```
 
-## Array properties
-
-Arrays, like strings, have a `length` property.
-
-You can check this by starting a node console in your terminal.
-
-```sh
-$ node
-> var arr = [1, 2, 3];
-undefined
-> arr
-[1, 2, 3]
-> arr.length
-3
-```
-
-## Array Getters and Setters
-
-You can **get** a single value out of an array using **bracket notation**.
-
-```sh
-$ node
-> var ingredients = ["Flour", "Water", "Salt"];
-undefined
-> ingredients[0]
-Flour
-> ingredients[1]
-Water
-> ingredients.length
-3
-```
-
-Did you notice how we use `[0]` to get the first value? In programming we count starting at zero.
-
-> The number inside of the brackets is called an **index**. Index just means the position of the item within the array.
-
-You can also **set** a value using bracket notation and an assignment operator (`=`).
+You can access elements in an array using the **index** of an element with **bracket notation** (remember: all arrays start at position 0! So to access the first element in an array, you need to access index 0!)
 
 ```js
-var scores = [80, 41, 47];
+let students = ["Ahmed", "Maria", "Atanas", "Nahidul", "Jack"]
 
-scores[2] = 29; // Change the last score
-scores[3] = 51; // Add a new score
+students[0] // Ahmed
+students[3] // Nahidul
 ```
 
-## Array methods
-
-Do you remember how strings have special functions called methods? Don't worry if not! Here's an example to jog your memory:
-
-```sh
-$ node
-> var name = "Daniel"
-undefined
-> name.toLowerCase()
-daniel
-```
-
-Arrays also have several methods that you can use.
-
-### `.sort()`
-
-_An array method that sorts the values in an array into ascending alphabetical or numerical order._
+You can also _assign_ new values to parts of an array:
 
 ```js
-var unorderedLetters = ["z", "v", "b", "f", "g"];
-var orderedLetters = unorderedLetters.sort();
-
-var unorderedNumbers = [8, 5, 1, 4, 2];
-var orderedNumbers = unorderedNumbers.sort();
+let students = ["Ahmed", "Maria", "Atanas", "Nahidul", "Jack"]
 
-console.log(orderedLetters); // logs [ 'b', 'f', 'g', 'v', 'z' ]
-console.log(unorderedLetters); // logs [ 'b', 'f', 'g', 'v', 'z' ]
+students[2] = "Bianca"
 
-console.log(orderedNumbers); // logs [ 1, 2, 4, 5, 8 ]
-console.log(unorderedNumbers); // logs [ 1, 2, 4, 5, 8 ]
+console.log(students) // ["Ahmed", "Maria", "Atanas", "Nahidul", "Jack"]
```

Comment:
@sansaid  there is a typo here in the output example the element in index number `2` should have the value of `Bianca ` not `Atanas`

Reply:
Thanks Elamin! I'll change that now.